src/storage/BaseProcessor.inl

```diff
@@ -83,5 +83,29 @@ void BaseProcessor<RESP>::doRemoveRange(GraphSpaceID spaceId,
         });
 }
 
+template<typename RESP>
+kvstore::ResultCode BaseProcessor<RESP>::doRange(GraphSpaceID spaceId,
+                                                 PartitionID partId,
+                                                 std::string start,
+                                                 std::string end,
+                                                 std::unique_ptr<kvstore::KVIterator>* iter) {
+    return kvstore_->range(spaceId, partId, start, end, iter);
+}
+
+template<typename RESP>
+kvstore::ResultCode BaseProcessor<RESP>::doPrefix(GraphSpaceID spaceId,
+                                                  PartitionID partId,
+                                                  std::string prefix,
+                                                  std::unique_ptr<kvstore::KVIterator>* iter) {
+    return kvstore_->prefix(spaceId, partId, prefix, iter);
+}
+
+template<typename RESP>
+kvstore::ResultCode BaseProcessor<RESP>::doRangeWithPrefix(
+        GraphSpaceID spaceId, PartitionID partId, std::string start, std::string prefix,
```

Comment:
const std::string &start, const std::string &prefix,

Reply:
It seems most interface in this class expect caller to use right value reference.